### DISCUSSION TITLE
Haven't recieved Benefits

### DISCUSSION THREAD

**Discussion by probablytaylors** [community/community#147894] [GitHub Education]
### Hi everyone,

Hi Everyone,

I haven't recieved my Benefits from Student Dev Pack I just got approved for. 
![image](https://github.com/user-attachments/assets/d797f2a7-6b5a-4b9a-990c-cc66a2e6af5a)

Is anyone having the same issue?

**Comment by davevad93** [comment]
Hi @Osintedx ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Osintedx was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by probablytaylors** [comment]
yeah its been more than 3 days.
